File: replace_url.py
import os

def replace_in_file(filepath):
    try:
        with open(filepath, 'r', encoding='utf-8') as f:
            content = f.read()
        if 'improving-ocean-jewelry-telephony.trycloudflare.com' in content:
            content = content.replace('improving-ocean-jewelry-telephony.trycloudflare.com', 'tourtravel.yber.in')
            with open(filepath, 'w', encoding='utf-8') as f:
                f.write(content)
            print(f"Replaced in {filepath}")
    except Exception as e:
        pass

for root, dirs, files in os.walk('.'):
    if 'node_modules' in root or '.git' in root or '.next' in root:
        continue
    for file in files:
        if file.endswith('.js') or file.endswith('.mjs') or file == '.env':
            replace_in_file(os.path.join(root, file))

File: replace_url.js
const fs = require('fs');
const path = require('path');

const OLD_URL = 'tourtravel.yber.in';
const NEW_URL = 'tourtravel.yber.in';

function replaceInFile(filePath) {
    try {
        let content = fs.readFileSync(filePath, 'utf8');
        if (content.includes(OLD_URL)) {
            content = content.split(OLD_URL).join(NEW_URL);
            fs.writeFileSync(filePath, content, 'utf8');
            console.log(`Replaced in ${filePath}`);
        }
    } catch (err) {
        console.error(`Error processing ${filePath}: ${err.message}`);
    }
}

function walkDir(dir) {
    const files = fs.readdirSync(dir);
    for (const file of files) {
        const fullPath = path.join(dir, file);
        if (file === 'node_modules' || file === '.git' || file === '.next') {
            continue;
        }
        const stat = fs.statSync(fullPath);
        if (stat.isDirectory()) {
            walkDir(fullPath);
        } else if (stat.isFile()) {
            if (file.endsWith('.js') || file.endsWith('.mjs') || file === '.env') {
                replaceInFile(fullPath);
            }
        }
    }
}

walkDir(__dirname);
